src/cpp/src/visual_language/embedding_model.hpp

```diff
@@ -29,12 +29,13 @@ class EmbeddingsModel {
                     const std::string& device,
                     const ov::AnyMap& properties);
 
+    EmbeddingsModel() = default;
```

Comment:
This probably comes from times when `EmbeddingsModel` was constructed in constructor body, so it can be removed now.

Reply:
the build fails without this here https://github.com/openvinotoolkit/openvino.genai/blob/e7038c1cc1bbc2a2d6623cf0cad765a7d0f85d58/src/cpp/src/visual_language/pipeline.cpp#L180